include/crypto/evp.h

```diff
@@ -58,21 +58,21 @@ struct evp_pkey_ctx_st {
 
         struct {
             EVP_KEYEXCH *exchange;
-            void *exchprovctx;
+            void *data;
```

Comment:
Should we move this 'data' outside the struct?

Reply:
No, let's leave it visually encapsulated